testing/trino-testing-containers/src/main/java/io/trino/testing/containers/TestContainers.java

```diff
@@ -55,4 +62,14 @@ public static String getPathFromClassPathResource(String resourcePath)
                 // and mounted from there.
                 .getResolvedPath();
     }
+
+    // utility method for setting exposed port to default values, like 5432 (postgres)
+    // for more comfortable usages in local environments for direct attach
```

Comment:
nit: We try to avoid comments which describe "what" is happening (unless the code is unreasonably complex - since the code is the "what") and instead try to focus  on "why this way?".